Automattic/jetpack
# Update newspack blocks 3.5.0 → 4.0.1

Fixes https://github.com/Automattic/wp-calypso/issues/69306
Fixes https://github.com/Automattic/wp-calypso/issues/73171

## Proposed changes:


Upgrading newspack blocks to 4.0.1
 * Mainly build and dependency changes
 * Also includes a fix to make the carousel and list posts blocks stop excluding the first post in the loop when used outside of a single posts/pages.

Improve the newspack sync system:
 * Fix issue with undefined constant
 * Updated readme with new instructions
 * Removed hardcoded version number in the npm command list, now users will get an error message unless they supply what to sync

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Create a simple site with a block theme
2. Update your sandbox as described in the comment below
3. Sandbox your site
4. Edit the template to replace any query block with a Blog Posts block
5. Look at the front-end of the site, it this block should include the configured number of posts starting with the most recent post.
6. Replace the Blog Posts block with a Post Carousel block
7. Repeat step 3.
8. Transfer your site to atomic
9. Use the instructions below to configure your atomic site to have this PR (use jetpack beta tester to change the `WordPress.com Features` branch.
10. Repeat steps 4-7
